articles/scp-3211/README.md

```diff
@@ -1,6 +1,6 @@
 # SCP-3211
 
-This is the sub-repository for [SCP-3211](https://www.scpwiki.com/scp-3211),
+This is the sub-repository for [SCP-3211](http://www.scpwiki.com/scp-3211),
```

Comment:
I'd prefer that PRs be kept on topic (there are also other places where URLs of this form would need to be changed for consistency)

Reply:
Sorry, yeah, noticed it and added it in. You should still add make this change due to the TLS issue.